browser/ui/browser_commands.cc

```diff
@@ -194,6 +194,7 @@ void CloseWalletBubble(Browser* browser) {
 }
 
 void CopySanitizedURL(Browser* browser, const GURL& url) {
+  LOG(ERROR) << "CopySanitizedURL";
```

Comment:
I assume this was forgotten before submitting to GitHub?

Reply:
yep, removed